content/en/docs/languages/sdk-configuration/declarative-configuration.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Declarative configuration
+linkTitle: Declarative configuration
+weight: 30
+---
+
+Declarative configuration uses a YAML file instead of environment variables.
+
+This approach is useful when:
+
+- You have many configuration options to set.
+- You want to use configuration options that are not available as environment
+  variables.
+
+## Supported languages
+
+The following OpenTelemetry SDKs support declarative configuration:
+
+- Java
```

Comment:
once the other PR is merged, we should update this to link to the java specific page